README.md

```diff
@@ -0,0 +1,21 @@
+# booster-server
+
+The REST API exposing operations to interact with the `vocabulary entries` and `notes`. 
+
+No concept of a user is supported for now, meaning that the system treats all requests as generic and does not differentiate between the requests issued by different users.
+
+### To start the application locally:
+* Start the PostgreSQL DB by running the following command:
```

Comment:
Maybe add the instruction with " ... by running the following command _from the project root_:"

Reply:
Fixed